src/Spago/Command/Fetch.purs

```diff
@@ -779,6 +788,9 @@ cycleError dep result = result
 -- | contexts.
 type ByEnv a = { core :: a, test :: a }
 
+mergeEnvs :: ∀ k v. Ord k => ByEnv (Map k v) -> Map k v
+mergeEnvs { core, test } = Map.union core test
+
```

Comment:
We are not using this anywhere else so I suggest inlining it in the `let` in line 629